brambling/tests/functional/test_invites.py

```diff
@@ -137,3 +145,15 @@ def test_context_data__no_invite(self):
         self.assertFalse(context['invited_person_exists'])
         self.assertFalse(context['sender_display'])
         self.assertFalse(context['invited_person_exists'])
+
+    def test_invite(self):
+        view = InviteAcceptView()
+        event = EventFactory()
+        view.invite = InviteFactory(content_id=event.pk, kind=Invite.EVENT, user=PersonFactory(first_name="Conan",last_name="O'Brien"))
+        view.content = view.invite.get_content()
+        view.request = RequestFactory().get('/')
+        view.request.user=PersonFactory()
+        self._add_session(view.request)
+        with mock.patch.object(wraps=Order.objects.for_request, target=Order.objects, attribute = 'for_request') as for_request:
+            view.handle_invite()
+        for_request.assert_called_once_with(create=True, request=view.request, event=view.content)
```

Comment:
This is great; we also should check the db state as long as we're here - i.e. that the order got created and has the expected properties.